### ISSUE TITLE
Emote Tabs

### ISSUE THREAD

**Issue by mtcdood** [TheUnlocked/Better-Mixer#10]
It might be necessary to make the menu scroll to see all emotes when a channel has a lot of emotes; alternatively a tabbed system might be user-friendly.

**Comment by TheUnlocked** [comment]
I'm surprised that it doesn't scroll by default like the preferences dialog does. I'll get on that.

**Comment by TheUnlocked** [comment]
Scrolling fixed in 8f45b37ba99fbd1f82f2793f439b61030b3b2726. I'm going to keep this open for the suggestion of tabs however.

**Comment by TheUnlocked** [comment]
I implemented a new categorization system for emotes. It's not tabs, but it should work well.

![image](https://user-images.githubusercontent.com/10186337/44557909-f7d61180-a6f5-11e8-8ab1-ba282e42e109.png)

**Comment by TheUnlocked** [comment]
@mtcdood
When I rewrote a big chunk of the injection code for the latest site update, I decided to not re-add categories both to fit better with the existing emote menu style and to make future fixes easier if Mixer changes their UI again. However, the part of the API which enables emote categories still exists so re-adding them would simply be an issue of updating the injection code.

I may add this feature back at some point, especially if I find an elegant way to do so (separating sub emotes would also be nice), but for now all of the emotes will stay in a single big list.
